src/components/TextButton/TextButton.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react';
+import style from './TextButton.st.css';
+import { TPAComponentsConsumer } from '../TPAComponentsConfig';
+import { ButtonNext } from 'wix-ui-core/button-next';
+import { ButtonProps as ButtonNextProps } from 'wix-ui-core/dist/src/components/button-next/button-next';
+
+export enum TEXT_BUTTON_PRIORITY {
+  primary = 'primary',
+  secondary = 'secondary',
+  link = 'link',
+}
+
+export interface TextButtonProps extends ButtonNextProps {
+  priority?: TEXT_BUTTON_PRIORITY;
```

Comment:
I almost forgot:
We need to add the `as` restriction.
We don't want to let users use elements other than `button` or `a`.
We should add a prop: `renderAs`, and pass this to the `button-next`'s `as` prop.
The reason for this, is that this is rendered in the viewer, and we need to follow SEO and A11Y standards.
Using only `button` and `a` will enforce this.

Reply:
I talked to Shlomi about this.
You can forget about this comment.
We'll leave it like this.